website/pages/community/style.css

```diff
@@ -1,4 +1,6 @@
 #p-community {
+  --brand-link: var(--nomad-link);
```

Comment:
Thanks for picking this up 💖 🧙 

I have a disgustingly minor nitpick ahah 😜 The underline is, unfortunately, not `text-decoration: underline` but an `::after` pseudo-element that uses the `--brand` property. And I think it's still blue.

So I think we should also set `--brand` to `--nomad-link` to fix that:

```suggestion
  --brand-link: var(--nomad-link);
  --brand: var(--nomad-link);
```

To be clear, I think this is a great temporary fix 0 even with the blue underline, I don't know that it's really that noticeable, it just bothered me 😬 

### Before this change

![blue-underline](https://user-images.githubusercontent.com/4624598/114254001-fa570980-997a-11eb-8ba3-5209f9482fa7.gif)

### After this change

![nomad-underline](https://user-images.githubusercontent.com/4624598/114254005-ffb45400-997a-11eb-8b6c-a2a7265adbb1.gif)

---

Also just a big +1 to fixing this in the component as you mentioned on [the Asana task](https://app.asana.com/0/1100423001970639/1200149091112826/f) by accepting a `product` prop. Added a note on the potential switch to `text-decoration: underline` as well 👍.

Reply:
Good catch! Thank you